cmd/preset.go

```diff
@@ -84,30 +88,30 @@ func (i *KoolInit) Execute(args []string) (err error) {
 	return
 }
 
-// NewInitCommand initializes new kool init command
-func NewInitCommand(init *KoolInit) (initCmd *cobra.Command) {
-	initCmd = &cobra.Command{
-		Use:   "init [PRESET]",
-		Short: "Initialize kool preset in the current working directory",
+// NewPresetCommand initializes new kool preset command
+func NewPresetCommand(preset *KoolPreset) (presetCmd *cobra.Command) {
+	presetCmd = &cobra.Command{
+		Use:   "preset [PRESET]",
+		Short: "Initialize kool preset in the current working directory. If none is specified, a wizard will be prompt to choose one",
```

Comment:
`If no preset argument is specified you will be prompted to pick among the existing options.`